_config.yml

```diff
@@ -33,6 +33,7 @@ imageNames:
   dikastes: calico/dikastes
   pilot-webhook: calico/pilot-webhook
   flexvol: calico/pod2daemon-flexvol
+  install-operator: quay.io/tigera/operator
```

Comment:
You should be find not adding this here. I'm going to raise a PR to remove this section within the day actually